### PR TITLE
feat: error state for homepage timeline

### PR DESCRIPTION
**Pull request by crfmc**
[Trello Ticket 238](https://trello.com/c/VZuVdSn3/238-error-states-for-homepage-timeline):
* Implement error state for homepage timeline

### CODE REVIEW

**Comment by crfmc** [comment]
[Error states for homepage timeline](https://trello.com/c/VZuVdSn3/238-error-states-for-homepage-timeline)

**Review by Bianca-Morris — APPROVED** [review]
Code looks good. Didn't test this locally, though, so if you are able to deploy it to devtest, I'd like to take a look at it there before merge. (I'd say, probably wait until after the Thursday deploy to merge this anyway, at this point, since there will be a lot of changes in that version of main by now, anyway.)

**Pull request merged**
